spec/models/teammate_spec.rb

```diff
@@ -1,4 +1,44 @@
 require 'rails_helper'
 
 describe Teammate do
+  describe ".ratings_count scopes count to current event" do
+    let(:green_event) { create(:event, name: "Green or Greener Event") }
+    let(:apple_event) { create(:event, name: "Most Apple Event") }
+
+    let(:dark_green_proposal) { create(:proposal, event: green_event) }
+    let(:light_green_proposal) { create(:proposal, event: green_event) }
+    let(:jack_proposal) { create(:proposal, event: apple_event) }
+    let(:withdrawn_proposal) { create(:proposal, event: green_event, state: "withdrawn") }
+
+    let(:guy_user) { create(:user) }
+    let(:gal_user) { create(:user) }
+
+    let(:green_guy_reviewer) { create(:teammate, event: green_event, user: guy_user, role: "reviewer") }
+    let(:apple_guy_reviewer) { create(:teammate, event: apple_event, user: guy_user, role: "reviewer") }
+    let(:green_gal_reviewer) { create(:teammate, event: green_event, user: gal_user, role: "reviewer") }
+
+    let!(:rating_1) { create(:rating, proposal: dark_green_proposal, user: guy_user) }
```

Comment:
Maybe create the ratings inside the actual it-block itself so that you can demonstrate the functionality with different combinations of ratings and events as you go.